### DISCUSSION TITLE
Is there a way to enable polling data from table?

### DISCUSSION THREAD

**Discussion by studiowizjo** [filamentphp/filament#1442] [Help]
Livewire components have feature to enable polling data, so user can view most current data. Is it possible to enable it to the Filament table?

**Comment by ghost** [comment]
Same question here. Thanks.

**Reply by studiowizjo** [reply]
I got idea to use Livewire events to achieve that.

You could use on **CreateRecord** page hook:

```
public function afterCreate()
{
    $this->emit('recordChanged');
}
```

similiar on **EditRecord**:

```
public function afterSave()
{
    $this->emit('recordChanged');
}
```

and then listen to it and refresh data on **ListRecords**:

```
protected $listeners = ['recordChanged' => '$refresh'];
```

Unfortunately this doesn`t work. Also for **Resource** page and some others I tried to.

**Comment by danharrin** [comment]
I think that polling tables would cause other issues you haven't considered, like filter dropdowns randomly closing, modals being cancelled etc. Unless the Alpine state is being preserved by Livewire?

**Comment by studiowizjo** [comment]
You are right, but can we refresh not whole component, but only records? And what about using Livewire emit system like I asked above? Do you have any sugesstions why it is not working?

**Reply by danharrin** [reply]
I don't, this is something you will need to figure out yourself. I can't see why the page wouldnt be displaying the latest data all the time anyway

**Reply by studiowizjo** [reply]
If you mean that you dont see the reason why data should be automatically refreshed to show most current data in table, there could be many reasons. In my example I`m building inventory system. Employers goes on storage with bar code scanners and scans products. At the same time, person who controll the process want to see progress which is now displayed in table. Another example could we preview of progress with scrapping something or any situation when data frequently changes (money exchange rates, and so on...).

Now, if you mean that you dont have idea why Filament pages cannot listen do Livewire events, that`s ok. 

Anyway, thanks for your time and help :).

**Reply by danharrin** [reply]
I meant that Livewire events should work fine with Filament, I understand why you want polling
